foxtrot-core/src/main/java/com/google/common/base/Stopwatch.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
```

Comment:
Hbase uses a dodo version of guava that breaks things all over the place.
New version of guava does not have the required constructor used by hbase.
That's why.

Also guava version should be enforced in dependencyManagement section in
the pom.
On 23-Apr-2016 10:55 pm, "Rishabh" <notifications@github.com> wrote:

In foxtrot-core/src/main/java/com/google/common/base/Stopwatch.java
<https://github.com/Flipkart/foxtrot/pull/134#discussion_r60831828>:

> @@ -0,0 +1,238 @@
> +/*
> + * Copyright (C) 2008 The Guava Authors

Any particular reason this class is copied into codebase ?

—
You are receiving this because you were mentioned.

Reply to this email directly or view it on GitHub
<https://github.com/Flipkart/foxtrot/pull/134/files/4c2f14c70d2f1f9782e2efb1c39de38babde6a8f#r60831828>